db/migrate/20211205115025_change_preference_allow_gifts_default.rb

```diff
@@ -0,0 +1,49 @@
+class ChangePreferenceAllowGiftsDefault < ActiveRecord::Migration[5.2]
+  def up
+    if Rails.env.staging? || Rails.env.production?
+      database = Preference.connection.current_database
+
+      puts <<~PTOSC
+        Schema Change Command:
+
+        pt-online-schema-change D=#{database},t=preferences \\
+          --alter "ALTER allow_gifts SET DEFAULT 0" \\
```

Comment:
I'm pretty sure this is right, since [the documentation](https://www.percona.com/doc/percona-toolkit/2.0/pt-online-schema-change.html#options) says everything that comes after the initial `ALTER table`, but it feels wrong.

Reply:
Should be `ALTER COLUMN allow_gifts SET DEFAULT 0` (https://stackoverflow.com/a/27907555).

Reply:
Ah, that makes it seem less wrong, thanks!